e2e/specs/drug-orders.spec.ts

```diff
@@ -1,120 +1,44 @@
 import { expect } from '@playwright/test';
 import { type Visit } from '@openmrs/esm-framework';
-import { generateRandomPatient, deletePatient, type Patient, startVisit, endVisit } from '../commands';
+import {
+  generateRandomPatient,
+  deletePatient,
+  generateRandomDrugOrder,
+  deleteDrugOrder,
+  createEncounter,
+  deleteEncounter,
+  getProvider,
+  type Patient,
+  startVisit,
+  endVisit,
+} from '../commands';
+import { type Encounter, type Provider } from '../types';
+import { type Order } from '@openmrs/esm-patient-common-lib';
 import { test } from '../core';
-import { MedicationsPage } from '../pages';
+import { MedicationsPage, OrdersPage } from '../pages';
 
 let patient: Patient;
 let visit: Visit;
+let drugOrder: Order;
+let encounter: Encounter;
+let orderer: Provider;
+let url: String;
 
 test.beforeEach(async ({ api }) => {
   patient = await generateRandomPatient(api);
   visit = await startVisit(api, patient.uuid);
+  orderer = await getProvider(api);
+  encounter = await createEncounter(api, patient.uuid, orderer.uuid);
+  drugOrder = await generateRandomDrugOrder(api, patient.uuid, encounter.uuid, orderer.uuid);
 });
 
-test('Record, edit and discontinue a drug order', async ({ page }) => {
-  const medicationsPage = new MedicationsPage(page);
+test('Edit and discontinue a drug order', async ({ page }) => {
```

Comment:
We do still need a test that covers recording a drug order, even if editing and deleting are done separately.

Reply:
sure done @ibacher